docs/gsoc/2020/ideas.rst

```diff
@@ -5,13 +5,13 @@ Introduction
 ------------
 
 This is the Google Summer of Code 2020 (GSoC'20) ideas page for ``uarray``,
-``unumpy`` and ``udiff``. The ``uarray`` library is is a backend mechanism
```

Comment:
Can you revert just this file, I'm pretty sure we aren't allowed to change it after GSoC begins and we get our ideas in.

Reply:
Sure.

Reply:
@hameerabbasi : I think these changes are fine.

Reply:
The changes themselves are fine, it's just that we've submitted our organization ideas and we risk disqualification if we modify the ideas now.

Reply:
@python-gsoc Could you comment on this?

Reply:
ok.